src/api/ledger/parse/account-order.js

```diff
@@ -1,14 +1,27 @@
 'use strict';
-const parseOrderBase = require('./order-base');
+const utils = require('./utils');
+const flags = utils.core.Remote.flags.offer;
+const parseAmount = require('./amount');
 
 // rippled 'account_offers' returns a different format for orders than 'tx'
```

Comment:
That can't be right.  I can see transaction vs ledger entry having different flags, but the metadata format is basically the same as ledger entry, only modified to show deltas.  You're getting Flags from the NewFields/FinalFields etc ?